### ISSUE TITLE
Not working with postcss-js?

### ISSUE THREAD

**Issue by rtsao** [MohammadYounes/rtlcss#48]
It seems rtlcss isn't working with postcss-js:

```javascript
var postcss = require('postcss-js');
var rtlcss = require('rtlcss');
var autoprefixer = require('autoprefixer');

var process = postcss.sync([ rtlcss, autoprefixer ]);

var css = {'.foo': {
  borderColor: 'red green blue black',
  left: '5px',
  transition: 'width 5s'
}};

console.log(process(css));
/*
{'.foo': {
  borderColor: 'red green blue black',
  left: '5px',
  WebkitTransition: width 5s ease,
  transition: 'width 5s ease'
}}
*/
```
Example:
http://requirebin.com/?gist=df4a18c20ba922ad02c2

**Comment by MohammadYounes** [comment]
You need to create an rtlcss instance as:

```js
var process = postcss.sync([ rtlcss(), autoprefixer ]);
```
Then I need to fix an issue caused by not having a source file, which will cause an exception [here](https://github.com/MohammadYounes/rtlcss/blob/master/lib/util.js#L64-L67)

Give me a minute to publish a fix.

Thanks!

**Comment by MohammadYounes** [comment]
Fix released in 1.7.2
